packages/@aws-cdk/aws-dynamodb-global/test/integ.dynamodb.global.expected.json

```diff
@@ -1,4 +1,6 @@
 [
+  {
+  },
```

Comment:
What’s going on here?

Reply:
This is empty, because I added a `Stack` to the test which doesn't contain any resources. If you think it will be clearer, I can add something like a `WaitConditionHandle` to it and add it to the expectations.

Reply:
oh, interesting...

Reply:
That's fine.